src/server/server_family.cc

```diff
@@ -549,6 +550,17 @@ void ServerFamily::Init(util::AcceptServer* acceptor, std::vector<facade::Listen
       LOG(WARNING) << "Invalid snapshot time specifier " << save_time;
     }
   }
+
+  string snapshot_cron_exp = GetFlag(FLAGS_snapshot_cron);
```

Comment:
I think that it should be an error to set both `save_schedule` _and_ `snapshot_cron`. As such, I would `CHECK()` and crash the process if they are both set (but not if both are empty).

Reply:
I agree with @chakaz  though not sure about the necessity of crashing. Maybe issuing a warning that `save_schedule` will be ignored.

Reply:
I usually go for `LOG(ERROR) << "ERROR"; exit(1)` so at least we don't have a stacktrace. But I agree with @chakaz that configuration mistakes should usually not be ignored

Reply:
absl has `QFATAL` (or `QCHECK`) for these cases exactly, but we don't have them as we're using glog :(
Ignoring `save_schedule` is also ok, but I would actually lean towards crashing :shrug:

Reply:
crashing or exit?

Reply:
Ideally - cleanly exiting :)
But really, it doesn't matter greatly. A warning is almost as good (although can be missed). I would not run both schedules in parallel though.

Reply:
I agree - we should not run both. So let's agree on error + exit.